src/modules/authentication/domain/person-permissions.ts

```diff
@@ -0,0 +1,30 @@
+import { RolleID } from '../../../shared/types/index.js';
+import { Person } from '../../person/domain/person.js';
+import { Personenkontext } from '../../personenkontext/domain/personenkontext.js';
+import { DBiamPersonenkontextRepo } from '../../personenkontext/persistence/dbiam-personenkontext.repo.js';
+
+export class PersonPermissions {
+    private personenkontexts?: Personenkontext<true>[];
+
+    private personInternal: Person<true>;
+
```

Comment:
Im Ticket steht "Dieses Aggregate verweist zunächst auf eine Person (per ID)"

Hier wird allerdigs das gesamte Person Aggregate als interner State gehalten. Ich finde das in diesem Usecase durchaus Sinnvoll, jedoch meine ich mich zu erinnern, dass wir mal festgelegt hatten das Aggregates keien anderen Aggregates halten sollen sondern nur deren Ids...

Reply:
Das stimmt

Reply:
Lustigerweise ist der REST von dem Aggregate auch ID-bezogen, das heißt, wenn wir uns da nur die id reinlegen müssen wir nur den Getter neu machen